packages/main/vite.config.js

```diff
@@ -51,7 +51,6 @@ const config = {
       external: [
         'electron',
         'chokidar',
-        '@kubernetes/client-node',
```

Comment:
highly recommended: I would remove this part from this pull request as it's not directly related to 1.0.0-RC7 to 1.0.0

Reply:
I think it is related, as the library has been changed to ESM between the 2 versions.

Reply:
but doing it there and now can have side consequences in production mode.

so it's always preferable to not mix two things at a single time

Reply:
OK, I've created https://github.com/podman-desktop/podman-desktop/pull/10439

Reply:
So this change should be removed ?

Reply:
yes, it will be removed after rebase